frontend/components/timer/index.jsx

```diff
@@ -20,6 +32,8 @@ const Timer = ({ time = 180, onTimeIsUp }) => {
       <div className='card'>
         <span>{seconds} s</span>
         <p>Timer component</p>
+        <input value={userTime} onChange={handleChange}  type="number" />
+        <button onClick={handleSubmit}>Set Time</button>
```

Comment:
In terms of accessibility, we can have our `input` and `button` wrapped in a `form` element with an `onSubmit` handler:
```
<form onSubmit={handleSubmit}>
    <input ref={userTime} type="number" />
    <button type="submit">Set Time</button>
</form>
```

Reply:
Perfect!, I did the changes already